packages/fxa-shared/subscriptions/types.ts

```diff
@@ -1,19 +1,22 @@
-export type PlanInterval = 'day' | 'week' | 'month' | 'year';
+import { Stripe } from 'stripe';
+
+export type PlanInterval = Stripe.Plan['interval'];
 
 export interface RawMetadata {
   [propName: string]: any;
 }
 
 export interface Plan {
+  amount: number | null;
+  currency: string;
+  interval_count: number;
+  interval: PlanInterval;
   plan_id: string;
   plan_metadata?: RawMetadata;
+  plan_name: string;
   product_id: string;
-  product_name: string;
   product_metadata?: RawMetadata;
-  currency: string;
-  amount: number;
-  interval: PlanInterval;
-  interval_count: number;
+  product_name: string;
 }
 
 // https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=COPS&title=SP+Tiered+Product+Support#SPTieredProductSupport-MetadataAgreements
```

Comment:
This URL changed to [ecosystem docs](https://mozilla.github.io/ecosystem-platform/docs/fxa-engineering/fxa-sub-platform#product-metadata) too, maybe?